css-paint-api/Overview.bs

```diff
@@ -451,8 +451,9 @@ Note: The user agent can optionally defer drawing images which are <a>paint-inva
 </div>
 
 The <a>draw a paint image</a> function is invoked by the user agent during the <a>object size
-negotiation</a> algorithm which is responsible for rendering an <<image>>, with |concreteObjectSize|
-set to the <a>concrete object size</a> of the <a>box</a>.
+negotiation</a> algorithm which is responsible for rendering an <<image>>, with |snappedConcreteObjectSize|
+set to the rounded <a>concrete object size</a> of the <a>box</a>, divided by the
```

Comment:
I don't think this phrasing captures pixel snapping correctly. Also, AFAICT, concrete object size is in CSS pixels. Rounding the concrete object size would suggests that
* this only affects objects whose size is non-integral - this is not true as it affects integral sizes when they are rastered into a non-integral transformed space (i.e. non-integer devicePixelRatio).
* we always snap an object of a particular size in the same direction, but it actually depends on the object's location.
* all browsers snap the same way. This seems to be the case but should probably not be required.
Instead, I think we should specify this in a way that allows the browser to use its own subpixel snapping determination.